src/app/pages/partners/partners.component.scss

```
@@ -0,0 +1,130 @@
+@media (max-width: 767px) {
+  .partner-logos--two-rows-mobile {
+    @apply flex-wrap justify-center pb-0;
+    overflow-x: visible;
+    gap: var(--partner-mobile-gap);
+  }
+
+  .partner-logos--two-rows-mobile .partner-logo-link {
+    flex: 0 0
+      calc(
+        (
+            100% - (var(--partner-mobile-columns) - 1) *
```

Stylelint (17.7.0)

[error] Unexpected newline after "*" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)